README.md

```diff
@@ -7,3 +7,67 @@
 PolicyEngine US is a microsimulation model of the US state and federal tax and benefit system.
 
 To install, run `pip install policyengine-us`.
+
+## START
```

Comment:
Lets make a section on:

How to contribute which is all from creating an issue to publishing a PR and requesting reviews